### ISSUE TITLE
Parse error on import list

### ISSUE THREAD

**Issue by jasonstolaruk** [ndmitchell/hlint#248]
HLint v1.9.32 (compiled w/ GHC 8.0.1) reports the following error, but the code compiles fine and I can't see what the issue is.

```
./lib/Mud/Cmds/Pla.hs:85:41: Error: Parse error: group
Found:
    import Data.IntMap.Lazy ((!))
    import Data.Ix (inRange)
  > import Data.List ((\\), delete, foldl', group, intercalate, intersperse, nub, nubBy, partition, sort, sortBy, unfoldr, zip4)
    import Data.List.Split (chunksOf)
    import Data.Maybe (fromJust, fromMaybe, isNothing)
```

Full source code [here.](https://github.com/jasonstolaruk/CurryMUD/blob/master/lib/Mud/Cmds/Pla.hs)

What do you think?

**Comment by ndmitchell** [comment]
The problem is you use the TransformListComp extension and the word group as a function and keyword. GHC has weird rules to disambiguate the words. The parsing library I use gets it a bit wrong. Is TransformListComp actually used in that file?

**Comment by jasonstolaruk** [comment]
I was only using the `TransformListComp` extension in one place. I stopped using it and now hlint doesn't report and errors.
Thanks!

**Comment by ndmitchell** [comment]
Great. My inclination is to close this ticket. It's an issue for the upstream Haskell-src-exts library, but in truth the TransformListComp is so rarely used I don't think anyone will bother fixing it.
